pkg/scheduler/plugins/reservation/rpod.go

```diff
@@ -0,0 +1,74 @@
+/*
+Copyright 2022 The Koordinator Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package reservation
+
+import (
+	corev1 "k8s.io/api/core/v1"
+
+	apiext "github.com/koordinator-sh/koordinator/apis/extension"
+	schedulingv1alpha1 "github.com/koordinator-sh/koordinator/apis/scheduling/v1alpha1"
+)
+
+var (
+	// AnnotationReservePod indicates whether the pod is a reserved pod.
+	AnnotationReservePod = apiext.SchedulingDomainPrefix + "/reserve-pod"
+	// AnnotationReservationNode indicates the node name if the reservation specifies a node.
+	AnnotationReservationNode = apiext.SchedulingDomainPrefix + "/reservation-node"
+)
+
+// GetReservePod returns a fake pod set as the reservation's specifications.
+// The reserve pod is only visible for the scheduler and does not make actual creation on nodes.
+func GetReservePod(r *schedulingv1alpha1.Reservation) *corev1.Pod {
```

Comment:
The name `GetReservePod` is ambiguous, NewReservationPod may be better.

Reply:
done